app/src/pages/inside/testCaseLibraryPage/expandedOptions/folder/folder.tsx

```diff
@@ -108,32 +112,34 @@ export const Folder = ({ folder, setActiveFolder, setAllTestCases, activeFolder
           <span className={cx('folders-tree__item-title--text')} title={folder.name}>
             {folder.name}
           </span>
-          <button
-            className={cx('folders-tree__tools', {
-              'folders-tree__tools--shown': areToolsShown,
-            })}
-            onClick={(event) => {
-              event.preventDefault();
-              event.stopPropagation();
+          {!!toolItems.length && (
```

Comment:
Please use `isEmpty` from lodash
```suggestion
          {!isEmpty(toolItems.length) && (
```